internal/store/postgres/scheduler/replay_repository.go

```diff
@@ -252,6 +254,30 @@ func (r ReplayRepository) UpdateReplay(ctx context.Context, id uuid.UUID, replay
 	return r.updateReplayRuns(ctx, id, runs)
 }
 
+func (r ReplayRepository) GetReplayTaskConfigByScheduledAt(ctx context.Context, scheduledAt time.Time) (map[string]string, error) {
```

Comment:
Any reason why we are not also filtering with the job name?

Reply:
It should use job name as well, will create a patch on it